docs/architecture/porting-existing-aspnet-apps/index.md

```diff
@@ -0,0 +1,82 @@
+
+Participants and Reviewers:
+
+> **Nish Anil**, Senior Program Manager, .NET team, Microsoft
+
+> **Mike Rousos**, Principal Software Engineer, .NET team, Microsoft
+
+> **Scott Addie**, Senior Content Developer, .NET team, Microsoft
+
+## Version
+
+This guide has been written to cover **.NET Core 3.1** version along with many additional updates related to the same “wave” of technologies (that is, Azure and additional third-party technologies) coinciding in time with the .NET Core 3.1 release. Updating from .NET Core 3.1 to .NET 5.0 (the next version) is relatively straightforward and certainly will require substantially less effort than porting from .NET Framework to .NET Core. Migrating from .NET Framework 4.x to .NET 5.0 will be similar to migrating to .NET Core 3.1. For more information, see [choosing the right .NET Core version](choose-net-core-version.md).
+
+## Who should use this guide
+
+The audience for this guide is mainly developers, development leads, and architects who are interested in learning how to move their existing apps written for ASP.NET MVC and Web API (.NET Framework 4.x) to .NET Core. ASP.NET Web Forms developers may also benefit from this guide, but should be sure to read [Blazor for ASP.NET Web Forms Developers](https://docs.microsoft.com/dotnet/architecture/blazor-for-web-forms-developers/) e-book as well.
+
+A secondary audience is technical decision-makers who are making plans for when to move their apps to .NET Core.
+
+Note that the target audience for this book is .NET developers with large, existing apps that run on ASP.NET MVC and Web API. Apps built on ASP.NET Web Forms are outside of the focus of this book, though much of the information comparing .NET Framework and .NET Core may still be relevant.
+
+## How you can use this guide
+
+You can read this book straight through, as we expect many readers to do. This will provide you first with considerations for whether you should port your app at all, followed by architectural differences between .NET and .NET Core. From there you'll learn strategies for migrating a large solution over time, and how to port a real app. Next, the book includes deployment scenarios that may help you address the need to run different apps while appearing as a single app to users. The book concludes with two case studies describing real apps that have made the move from ASP.NET MVC to ASP.NET Core.
+
+Whether or not you choose to start from the first chapter, you can also refer (back) to any of these chapters to learn about specific topics:
+
+- [Architectural differences](architectural-differences.md)
+- [Migrate large solutions](migrate-large-solutions.md)
+- [Sample migration](example-migration-eshop.md)
+- [Deployment scenarios](deployment-scenarios.md)
+
+This guide is available both in PDF (TODO: add link) form and online. Feel free to forward this document or links to its online version to your team to ensure a common understanding of these topics.
```

Comment:
@nishanil Do you already have an aka.ms link for the PDF?

Reply:
Not yet. We can generate one and create an aka.ms link. @sughosneo can you help?

Reply:
Sure @nishanil .

Reply:
@sughosneo Please let me know once you have the link. I can add it here.

Reply:
Hi @scottaddie - here is the link https://aka.ms/aspnet-porting-ebook for the PDF.

Reply:
@scottaddie can you update the link in a new commit please?

Reply:
Yes, I'll take care of it.